.github/workflows/goreportcard.yaml

```diff
@@ -0,0 +1,52 @@
+name: Go Report Card
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
We don't have a "master" branch so this can be removed
```suggestion
```

Reply:
I was referring golangci-lint workflow and wondering about it. Updated it. Thanks!